test/perf/perf.test.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable no-undef */
+/* eslint-disable camelcase */
+const { fetch } = require('@adobe/helix-fetch');
+const { getUrls, parseTiming } = require('../utils');
+
+const baseService = process.env.HLX_FASTLY_NAMESPACE;
+const testService = process.env.TEST_SERVICE;
+/**
+ * generates strings with information about changes in rendering time
+ *
+ * @param {*} timeObj1 object of pipeline step times and desc
```

Comment:
```suggestion
 * @param {object} timeObj1 object of pipeline step times and desc
```

Not sure what `@param {*}` does. Might be correct..